packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -59,13 +59,14 @@ class ReorderableListView extends StatefulWidget {
     @required this.onReorder,
     this.scrollDirection = Axis.vertical,
     this.padding,
+    this.canControlexpandedReorderList = false,
```

Comment:
nit: this parameter name seems a it confusing to me, and should probably have a capital `Expanded` anyway.

Also, please revert the formatting changes made in this file.